pyccl/__init__.py

```diff
@@ -62,7 +62,7 @@
 from .haloprofile import nfw_profile_3d, einasto_profile_3d, hernquist_profile_3d, nfw_profile_2d
 
 # Specific to massive neutrinos
-from .neutrinos import Omeganuh2, nu_masses
+from .neutrinos import Omega_nu_h2, Omeganuh2, nu_masses
```

Comment:
If you're asking why am I renaming this, it's because I find "Omeganuh2" extremely hard to read. Python universally adopts `snake_case` while C-like languages adopt `SwapCase`. Similar to how we have `halomod_power_spectrum` rather than `HalomodPowerSpectrum`.